media-sound/audacity/metadata.xml

```diff
@@ -5,16 +5,15 @@
     <email>proaudio@gentoo.org</email>
     <name>Gentoo ProAudio Project</name>
   </maintainer>
-  <maintainer type="person" proxied="yes">
```

Comment:
This change is _not_ the same thing. Having it here means that they're visible to bug wranglers and it's an indication to CC them on bugs. The `<upstream>` section does not achieve that.